src/util/env.ts

```diff
@@ -0,0 +1,37 @@
+// is ESM
+// https://github.com/sindresorhus/is-docker/blob/main/index.js
```

Comment:
this dep is ESM so I got compilation errors.
Can probably fixed with the right webpack config but copy pasting the code was just as fast sweat_smile